packages/@react-aria/searchfield/docs/useSearchField.mdx

```diff
@@ -106,28 +107,26 @@ function SearchField(props) {
 ## Styling
 
 This example uses CSS to reset the default browser styling for a search field and implement
-custom styles. It also includes a custom clear button, using props
-from <TypeLink links={docs.links} type={docs.exports.useSearchField} />
-and <TypeLink links={buttonDocs.links} type={buttonDocs.exports.useButton} />
-(from @react-aria/button).
+custom styles. It also includes a custom clear button, built with [useButton](useButton.html).
```

Comment:
this example jumps a bit in chrome when the clear button appears, maybe set a larger default height?